src/guide/kuzzle-backend-setup/aws-marketplace.md

```diff
@@ -4,4 +4,63 @@ algolia: true
 title: AWS Marketplace
 description: AWS Marketplace
 order: 1200
----
\ No newline at end of file
+---
+
+# AWS Marketplace
+<div class="alert alert-info">
+To follow this tutorial, you need a __valid AWS account__.
+</div>
+
+_In this guide, you'll learn where and how to use our AWS Marketplace AMI. It's a good way to test Kuzzle in a cloud environment. In addtion, we recommend you to use our [Kuzzle Admin Console](http://console.kuzzle.io), the most comfy way to play with Kuzzle._
```

Comment:
`In addtion` => `In addition`
`we recommend you to use` => `we recommend that you use` (or `we recommend you use`)
`most comfy way to play` (slang, and should be comfiest anyway) => `easiest way to play`